app/forms/new_phone_form.rb

```diff
@@ -148,6 +148,8 @@ def recaptcha_validator_args
     args = { recaptcha_version:, analytics: }
     if IdentityConfig.store.phone_recaptcha_mock_validator
       args.merge(validator_class: RecaptchaMockValidator, score: recaptcha_mock_score)
+    elsif FeatureManagement.recaptcha_enterprise?
+      args.merge(validator_class: RecaptchaEnterpriseValidator)
```

Comment:
❤️ I feel like this validates the choice to remove inheritance from the earlier PR